src/main/kotlin/org/partiql/scribe/targets/redshift/RedshiftAstToSql.kt

```diff
@@ -123,4 +131,52 @@ public open class RedshiftAstToSql(context: ScribeContext) : AstToSql(context) {
             else -> super.visitDataType(node, tail)
         }
     }
+
+    /**
+     * The PartiQL plan and thus AST will fully-qualify ORDER BY variable references and paths with an implicit
+     * binding tuple name for set operations. For Redshift, there is no implicit binding tuple name for the set ops. So
+     * we must remove the prefix binding tuple name from paths.
+     *
+     * For example, the query
+     *   (SELECT a FROM ...) UNION (SELECT a FROM ...) ORDER BY a
+     * will have an extra qualification
+     *   (SELECT a FROM ...) UNION (SELECT a FROM ...) ORDER BY "_1".a
```

Comment:
self-review: we remove some additional qualification that PartiQL's plan + AST (after transformation from plan -> ast) provides. The implicit alias causes errors in Redshift, Trino, and Spark, hence why we remove it here.